types/src/crypto/asymmetric_key.rs

```diff
@@ -586,8 +609,8 @@ pub enum Signature {
     // it is convenient and cheap to use `signature.as_ref()`.
     Ed25519([u8; ed25519_dalek::SIGNATURE_LENGTH]),
     /// secp256k1 signature.
-    #[data_size(skip)] // Manually verified to have no data on the heap.
-    Secp256k1(ecdsa::Signature),
+    #[data_size(skip)]
```

Comment:
Probably don't have to skip any more.

Reply:
Fixed in 899be2413bd24f04ad763b7eafba11fa4b9a33a6